src/core/viz/expressions/viewportFeatures.js

```diff
@@ -0,0 +1,75 @@
+import BaseExpression from './base';
+import * as schema from '../../../core/schema';
+import Property from './basic/property';
+
+/**
+ * Generates a list of features in the viewport
+ *
+ * For each feature, the properties specified as arguments to this expression will be available.
+ * Filtered features will not be present in the list.
+ * This expression cannot be used in the visualization, it can only be used in JavaScript code as in the example below.
```

Comment:
It cannot be used to render, and type-checking should enforce that, I find the first sentence confusing